Configuration/AlCa/python/autoPCL.py

```diff
@@ -11,6 +15,6 @@
            'PromptCalibProdEcalPedestals': 'EcalPedestals',
            'PromptCalibProdLumiPCC': 'LumiPCC',
            'PromptCalibProdPPSTimingCalib' : 'PPSTimingCalibration',
-           'PromptCalibProdPPSDiamondSampicTimingCalib' : 'PPSDiamondSampicTimingCalibration',
+           'PromptCalibProdPPSDiamondSampic' : 'PPSDiamondSampicTimingCalibration',
```

Comment:
I see that values in the `autoPCL` dictionaries are kept intact. Is it by purpose?

Reply:
please see answer in https://github.com/cms-sw/cmssw/pull/38209#issuecomment-1145373710